pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -976,8 +976,8 @@ private void synchronize(
             .p2pEnabled(p2pEnabled)
             .discovery(peerDiscoveryEnabled)
             .ethNetworkConfig(ethNetworkConfig)
-            .discoveryHost(discoveryHost)
-            .discoveryPort(discoveryPort)
+            .advertisedHost(discoveryHost)
+            .listenPort(discoveryPort)
```

Comment:
Is it weird to have the names mismatching here?

Reply:
Yep, thats weird :) . Updated the variable names.  Also updated the description of the corresponding options.